jquery.overlay.js

```diff
@@ -4,11 +4,13 @@
  * Repository: https://github.com/yuku-t/jquery-textoverlay
  * License:          MIT
  * Original Author:  Yuku Takahashi
- * Modifying Author: Niki Castle
+ * Modifying Authors: Niki Castle, Nick Willett-Jeffries
  *
- * This version is from https://github.com/neekee/jquery-overlay/tree/master, which includes
- * modifications made to get this working for input fields and resizeable textareas, and to
- * allow nested matching.
+ * This version is from https://github.com/nwj/jquery-overlay/tree/master, which introduces support for
```

Comment:
you also include support for custom css other than background-color (although does that screw up the overlay alignment in any way?